### DISCUSSION TITLE
Site map picture with locations

### DISCUSSION THREAD

**Discussion by averyhabbott** [netbox-community/netbox#14149] [Ideas]
It would be nice to be able to upload a picture for a site (satellite photo, floor plans, etc) and to then be able to drag and drop locations onto the map.  This could be used for locating closets at a site, OSP wiring plants, etc.

Clicking on one of the locations on the photo would take you to the NetBox page for that location.

Hovering the mouse over the list of locations would highlight it on the photo.

There could be a tab on the location page that showed the site image with the individual location highlighted (similar to how looking at a device shows its rack with the individual device highlighted).

**Comment by DanSheps** [comment]
This could be something you could suggest for the netbox-floorplan plugin.

**Reply by averyhabbott** [reply]
Potentially, BUT NetBox has already demonstrated that determining physical location for a site is core functionality.  This seems more like further development of that core functionality, extending it down to the location level and adding some form of map that isn't just a Google Maps link.

Maybe that means this is two-fold?  Extend grid coordinates to location, then add jpg/png mapping as part of the netbox-floorplan plugin?
